src/services/utils/index.ts

```diff
@@ -99,3 +100,72 @@ export async function verifyGitHubEmail(token: string, email: string) {
 
     return true;
 }
+
+
+const accountSid = import.meta.env.VITE_TWILIO_ACCOUNT_SID;
+const authToken = import.meta.env.VITE_TWILIO_AUTH_TOKEN;
```

Comment:
This is exposed to the public. If this token is needed for this, we should use a cloud function instead.